quickjs/src/main/jni/quickjs-jni.c

```diff
@@ -3,6 +3,7 @@
 #include <malloc.h>
 #include "quickjs/quickjs.h"
 
+// TODO(szurbrigg): delete these global refs on teardown.
```

Comment:
Should we store these in a struct next to the `JSRuntime` and return the struct instance pointer to the Java?

I originally had that setup for storing the `JSContext` and `JSRuntime` until I figured out that you can get one from the other.

Reply:
Definitely. We'll have other pointers to track as well when we start binding Java and JS objects.